frontend/src/script.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// fetch the secondary calendar with the schedule (just one right now, will be multiple later)
+var currCalendarIndex = 0; 
+var idArray;
+export async function fetchUserCalendar() {
+  // change back to /fullCalendar
+  const response = await fetch('/fullCalendar');
+  const responseJson = await response.json();
+  return responseJson;
+}
+
+// hide / show the appropriate elements on the page 
+function showButtons() {
```

Comment:
It doesn't seem like you're using all of these methods from the old google calendar display for full calendar - might want to delete the unnecessary ones if so

Reply:
This is weird, I had removed these functions... Maybe its on a different commit?